web/pf4/src/components/KChart.tsx

```diff
@@ -118,6 +118,14 @@ class KChart extends React.Component<KChartProps, State> {
       />
     );
     const scaleInfo = this.scaledAxisInfo(data);
+    const seriesBuilder = (this.props.chart.chartType === 'area')
```

Comment:
I sometimes tend to use the ternary like this. But, could we sacrifice shortness for clarity and use a `switch`? or, at least, include some indenting to make it clear the nesting?
Also, variable naming could be improved :)
Suggestion: line -> serie (and same for the keys)

Reply:
I've changed the indentation style which I think makes it much more readable.
I'm not a big fan of switch to just assign a variable, especially in ts/js